linkis-engineconn-plugins/spark/pom.xml

```diff
@@ -26,7 +26,7 @@
 
   <artifactId>linkis-engineplugin-spark</artifactId>
   <properties>
-    <spark.version>2.4.3</spark.version>
+    <spark.version>2.4.7</spark.version>
```

Comment:
There are many version upgrades and modifications, it is recommended to keep 2.4.3 for the time being